src/SURL.h

```diff
@@ -29,26 +29,27 @@ class SURL {
   SURL();
```

Comment:
[Optional] Consider using 2 commits in the same PR. One for .h and the other for .cpp.